Source/WebKit/WebProcess/Model/ModelProcessModelPlayer.cpp

```diff
@@ -115,6 +122,17 @@ void ModelProcessModelPlayer::setBackgroundColor(WebCore::Color color)
     send(Messages::ModelProcessModelPlayerProxy::SetBackgroundColor(color));
 }
 
+/// This comes from JS side, so we need to tell Model Process about it. Not to be confused with didUpdateEntityTransform().
+void ModelProcessModelPlayer::setEntityTransform(WebCore::TransformationMatrix transform)
+{
+    send(Messages::ModelProcessModelPlayerProxy::SetEntityTransform(transform));
+}
+
+bool ModelProcessModelPlayer::supportsTransform(WebCore::TransformationMatrix transform)
+{
+    return ModelProcessModelPlayerProxy::transformSupported(transform);
+}
```

Comment:
I'm not sure I understand this. Does this mean you're doing synchronous IPC?

Reply:
this is a static method. So WP side simply calls the RE lib math functions for the validation without invoking Model Process